builtin/providers/dns/data_dns_a_record_set_test.go

```diff
@@ -34,23 +36,30 @@ func TestAccDataDnsARecordSet_Basic(t *testing.T) {
 			[]string{
 				"129.6.15.30",
 			},
+			"time-c.nist.gov",
 		},
 	}
 
-	var steps []resource.TestStep
-
 	for _, test := range tests {
-		ts := resource.TestStep{
-			Config: test.DataSourceBlock,
-			Check: resource.ComposeTestCheckFunc(
-				testCheckAttrStringArray(fmt.Sprintf("data.dns_a_record_set.%s", test.DataSourceName), "addrs", test.Expected),
-			),
-		}
-		steps = append(steps, ts)
+		recordName := fmt.Sprintf("data.dns_a_record_set.%s", test.DataSourceName)
+
+		resource.UnitTest(t, resource.TestCase{
```

Comment:
1. didn't make sense to me that the data_cname_record_set test was a unit test, and this was an acceptance test
2. I can change it back, but it didn't seem to run when I ran `make test TEST=./builtin/providers/dns` ... may be I'm just missing something for how to run tests - I'm new to this world of go